inspirehep/modules/migrator/cli.py

```diff
@@ -37,6 +38,8 @@
     add_citation_counts,
     migrate,
     migrate_broken_records,
+    migrate_chunk,
+    split_marc,
```

Comment:
Importing a regexp means you're duplicating the logic that uses it, which then should be abstracted in a helper.